src/InstrumentationEngine.ProfilerProxy/dllmain.cpp

```diff
@@ -361,6 +361,8 @@ namespace ProfilerProxy
             CProxyLogging::LogError(_T("dllmain::DllGetClassObject - Failed to load profiler with error code 0x%x"), hr);
         }
 
+        CProxyLogging::Shutdown();
```

Comment:
> CProxyLogging::Shutdown() [](start = 7, length = 26)

Consider using an RAII structure so that logging gets shutdown if an exception occurs.